### PR TITLE
Remove redux from the project

### PR DESCRIPTION
**Pull request by DominykasSueris**
Redux removed from the project. useContext is in use there required.

### CODE REVIEW

**Comment by DominykasSueris** [comment]
Ref #37

**Comment by augustinasrce** [comment]
"...**where** required"

File: src/components/Input/Span.tsx
Comment (augustinasrce) [inline src/components/Input/Span.tsx:1]: Not used

**Comment by DominykasSueris** [comment]
Changes had been made where highlighted. Please review again @augustinasrce for merge to main branch.

File: src/services/aws/authSpecs.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+export enum AuthTarget {
+  aws = "Aws",
+  azure = "Azure",
+  gcloud = "gCloud"
+}
+
+export enum AuthRegion {
+  "us-east-1" = "united-states-east-1",
+  "us-east-2" = "united-states-east-2",
+  "us-east_2" = "us-east-2",
```
Comment (augustinasrce) [inline src/services/aws/authSpecs.ts:10]: I would use something more user-friendly for a particular label.

**Review by augustinasrce — DISMISSED** [review]

**Comment by DominykasSueris** [comment]
Please review last changes @augustinasrce.

**Review by augustinasrce — APPROVED** [review]

**Pull request merged**